tests/integration/Owncloud/OcisPhpSdk/ShareCreatedModifyTest.php

```diff
@@ -54,16 +54,33 @@ public function testDeleteIndividualShare(): void
         $this->fileToShare->invite($this->marie, $this->viewerRole);
         $shares = $this->ocis->getSharedByMe();
         foreach ($shares as $share) {
-            $this->assertInstanceOf(ShareCreated::class, $share);
+            $this->assertInstanceOf(
+                ShareCreated::class,
+                $share,
+                "Expected class to be " . ShareCreated::class
```

Comment:
I think we can just write that class name in the string like:
```suggestion
                "Expected class to be 'ShareCreated'"
```